content/events/2023/01/2023-01-05-uswds-monthly-call-january-2023.md

```diff
@@ -0,0 +1,43 @@
+---
+title: USWDS Monthly Call - January 2023
+deck: Answering your most-asked questions about the U.S. Web Design System
+kicker: USWDS
+summary: Join the Design System team as we tackle your most commonly-asked questions.
+host: U.S. Web Design System
+event_organizer: Digital.gov
+cop_events: ""
+registration_url: https://gsa.zoomgov.com/meeting/register/vJItceqrrDsuEhxA7eoRggB3Cf4RRjcrMjU
+captions: https://www.streamtext.net/player?event=BIS-GSA-JY
+date: 2023-01-19 14:00:00 -0500
+end_date: 2023-01-19 15:00:00 -0500
+# See all topics at https://digital.gov/topics
+topics:
+  - uswds
+  - open-source
+  - usability
+slug: uswds-monthly-call-january-2023
+# zoom, youtube_live, adobe_connect, google
+event_platform: zoom
+primary_image: 2023-uswds-monthly-call-jan-title-card
+---
+Join the Design System team as we tackle your most commonly-asked questions.
+
+In this session, we’ll answer questions like:
+
+* Can and should I theme USWDS?
+* What's the component testing process?
+* How do I set custom fonts?
+* How do I use USWDS with my technology stack?
+* How do I update my project setup after moving to USWDS 3?
+* When should I expect updates to the USWDS design kit?
+* When will new components be released?
+
+**This event is best suited for:** Designers and developers who use the U.S. Web Design System plus those who are planning to use it.
+
+## Speakers
+
+* **Dan Williams -** Product Lead, USWDS
+* **James Mejia -** Engineer, USWDS Contractor
+* **Amy Leadem -** Front-End Developer, USWDS Contractor
+* **Bonnie Cameron -** Title, USWDS Contractor
+* **Jacline Contrino -** Title, USWDS Contractor
```

Comment:
@jdotyoon should we add the Bonnie and Jacline's titles here?

Reply:
Yes! Thanks for catching that! Would you know their titles?